README.md

```diff
@@ -1 +1,38 @@
-# jitsi-meet-web-sdk
\ No newline at end of file
+# Jitsi Meet Web SDK
+The Jitsi Meet Web SDK provides the same user experience as the [Jitsi Meet](https://github.com/jitsi/jitsi-meet) app, in a customizable way which you can embed in your apps.
+
+## Install
+```bash
+npm install jitsi-meet-websdk
+```
+### Modules
+#### initExternal
+To import the Jitsi Meet External API in a non-React project:
+```js
+// index.js
+initExternal().then((externalApi) => { ... });
+
+// index.html
+window.onload = () => {
```

Comment:
Why would one do this? It can all be handled in the JS file, right? Once `initExternal` resolves one can start writing the app logic already.